appcircle_cache_pull/1.0.4/component.yaml

```diff
@@ -0,0 +1,27 @@
+platform: Common
+buildPlatform:
+displayName: "Cache Pull"
+description: "Downloads cache from Appcircle, extracts files and folders to source locations."
+docsUrl: https://docs.appcircle.io/workflows/common-workflow-steps#cache-pull
+requiredComponents: 
+precedingComponents: "appcircle_git_clone"
+followingComponents: "appcircle_cocoapods,appcircle_ios_build_sign,appcircle_ios_build_simulator,appcircle_ios_build_for_testing"
```

Comment:
> [So, we should not include any preceding or required component for "Cache Push" based on the default values in "Included Paths".](https://github.com/appcircleio/appcircle-workflow-components/pull/352#issuecomment-1973043160)

@boztopuz  Onur gave the feedback above. Could you please make changes accordingly?